src/main/java/Truck.java

```diff
@@ -0,0 +1,9 @@
+class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck started working.");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck stopped working.");
+    }
```

Comment:
Similarly, the message in `System.out.println` should be more informative and indicate that it is a Truck that has stopped working. For example: `System.out.println("Truck has stopped working.");`